src/index.html

```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                            <li>Submit variations with sensitive clinical data to <a href="https://www.ebi.ac.uk/ega/" target="_blank">EGA</a>.</li>
                             <li>Submit variations with clinically relevant genetic variant data, i.e. data that relates genetic variation(s) with clinical significance values (e.g. pathogenic, benign, etc.), to the <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">ClinVar</a> archive at <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">NCBI</a>.</li>
                         </ul>
                     </div>
 
                     <h3 id="key-stages" class="padding-bottom-large">Key stages of EVA submissions</h3>
 
-                    <h4 id="submit-prepare"><span class="icon icon-generic" data-icon="'" aria-hidden="true"></span> Prepare</h4>
-                    <div>
-                        <ul>
-                            <li>Prepare <a href="?Help#submissionPanel&link=is-my-data-suitable" target="_blank">valid VCF file</a>(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide" target="_blank">EVA VCF validation suite</a>.</li>
-                            <li>Complete a <a href="files/EVA_Submission_template.METADATA_TEMPLATE_VERSION.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. Please provide as much metadata as possible since this information is extremely useful for downstream analysis and is directly related to the frequency at which datasets archived at EVA are reused. For reference, here is an <a href="files/EVA_Submission_template.METADATA_TEMPLATE_VERSION_mockup.xlsx" target="_blank"><b>example</b></a> of a completed metadata template.</li>
-                        </ul>
-                        <p>Please also note that the template requires the submitter to fill in some personal data, which will be used as described in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/eva-submission-service" target="_blank">privacy notice</a>.</p>
-                    </div>
-
-                    <h4 id="submit-contact"><span class="icon icon-generic" data-icon="C" aria-hidden="true"></span> Contact</h4>
-                    <div>
-                        <p>Contact  <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data.</p>
-                    </div>
-
-                    <h4 id="submit-submit"><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
-                    <div>
-                        <p>Upload your VCF file(s), metadata template and any associated data file(s) to your private FTP location.</p>
+                    <div class="submit-data-diagram">
+                        <div class="box green">
+                            <h3>📄 Prepare your data and metadata</h3>

```

Comment:
Personally I don't think the emojis look professional, I would use the same icon source that is used in the current submit page - we just need to find the appropriate additional icons for the new steps. If necessary, we can use the icons in the document diagram, just need to credit [flaticon.com]() somewhere as that's the source I used.